acquia/mc-cs-plugin-custom-objects
# MAUT-7884 Entity changes for migration for adding unique identifier column

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ No]
| New feature/enhancement? (use the a.x branch)      | [ Yes]
| Deprecations?                          | [ NA]
| BC breaks? (use the c.x branch)        | [ NO]
| Automated tests included?              | [ No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes # [MAUT-7884](https://backlog.acquia.com/browse/MAUT-7884l) 



#### Description:



#### Steps to test this PR:


1. Open this PR for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new custom object "Book", try adding fields. A new "is Unique Identifier" toggle button should be visible.
3. Add a name "book uuid" and toggle "is unique identifier to true.
4. Save the custom object.
5. Edit the custom object. Try editing the field "book uuid". The 'is unique identifier' and the 'required' toggle buttons should be readonly.
